schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/SchemaVersion.java

```diff
@@ -37,6 +39,14 @@ public SchemaVersion(String schemaText, String description) {
         this.schemaText = schemaText;
     }
 
+    public SchemaVersion(String schemaText, String description, Byte initialState) {
+        this.description = description;
+        this.schemaText = schemaText;
+        if(initialState != null) {
```

Comment:
Style of defaulting differs to other places such as https://github.com/satishd/registry-1/blob/669b66f1ce0597398c5dc277c7099b95dd9ac957/schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/SchemaVersionInfo.java

should follow standard pattern in default setting a value across the project, all ways are valid, but really helps with consistency of code